verification/api/handler.go

```diff
@@ -472,3 +475,124 @@ func sendChallengeResponseSessionCreated(c *gin.Context, id string, jsonSession
 	c.Header("Location", path.Join("session", id))
 	sendChallengeResponseSessionWithStatus(c, http.StatusCreated, jsonSession)
 }
+
+func (o *Handler) getKey() (jwk.Key, error) {
+	var key jwk.Key
+
+	protoKey, err := o.Verifier.GetPublicKey()
+	if err != nil {
+		return key, err
+	}
+
+	key, err = jwk.ParseKey([]byte(protoKey.Key))
+	if err != nil {
+		return key, err
+	}
+
+	return key, nil
+
+}
+
+func (o *Handler) getVerificationMediaTypes() ([]string, error) {
+	mediaTypes, err := o.Verifier.SupportedMediaTypes()
+	if err != nil {
+		return mediaTypes, err
+	}
+
+	return mediaTypes, nil
+
+}
+
+func (o *Handler) getVerificationServerVersion() (string, error) {
+	vtsState, err := o.Verifier.GetVTSState()
+	if err != nil {
+		return "", err
+	}
+	version := vtsState.ServerVersion
+	return version, nil
+
+}
+
+func (o *Handler) getVerificationServerStatus() (string, error) {
+	vtsState, err := o.Verifier.GetVTSState()
+	if err != nil {
+		return "", err
+	}
+	state := vtsState.Status.String()
+	return state, nil
+
+}
+
+func (o *Handler) getVerificationEndpoints() (*capability.ApiEndpoints, error) {
+	challengeResponseLink := "/challenge-response/v1/newSession"
```

Comment:
we need to make sure that these are kept in sync with the master copies in router.go.

It'd be ideal if we could source the information from the name of the corresponding handler function.